input/pagecontent/future.md

```diff
@@ -34,8 +34,18 @@ The approach to develop and mature AU Core is outlined below making reference to
+* **Composition**: Future versions of AU Core will develop and test AU Core Composition profile to support approaches to sharing clinical documents across multiple use cases.
+* **DocumentReference**: Future versions of AU Core will develop and test approaches and AU Core DocumentReference profile to support sharing of clinical notes across multiple use cases to support common implementation.
+* **RelatedPerson**: Development and testing of AU Core RelatedPerson profile to support the exchange of core digital health and administrative information that involves persons with a personal relationship or non-healthcare-specific professional relationship to the patient is planned for the AU Core R2 scope.
+* **HealthcareService**: Development and testing of AU Core HealthcareService profile to define minimum requirements for support health care service information across multiple use cases to support common implementation. Work will also be undertaken to consider guidance on resource selection across HealthcareService, PractitionerRole, Organization, and Location.
+* **Device**: Use of the Device resource for medical devices, and devices that perform, observe, or author clinical data, are required to support multiple downstream use cases including eRequesting and patient summary. Future versions of AU Core will develop and test profiles to support common implementation of Device information.
+* **Coverage**: Use of the Coverage resource is required to support multiple downstream use cases including eRequesting. Future versions of AU Core will develop and test minimum requirements for coverage across multiple use cases to support common implementation.
+* **ServiceRequest**: Use of the ServiceRequest resource is required to support multiple downstream use cases including eRequesting. Future versions of AU Core will develop and test minimum requirements for requests across multiple use cases to support common implementation.
+* **Diagnostic Results**: Additional AU Core profiles to support common implementation of additional aspects of diagnostic results including imaging, specimen, and reports will be developed and test in future versions of AU Core.
+* **Consideration whether some clinical elements should not allow Missing Data**: It is not clear at this point whether there is a case for prohibiting 'missing data' and 'suppressed data' on resource.code e.g. AllergyInterolance.code, Condition.code, or Medication.code. At this time it is not clear how prohibition could be structured in AU Core noting there is no equivalent obligation code for 'don't allow missing data' and as the data type for these elements is CodeableConcept, preadoption of R5 mustHaveValue is not applicable (only for primitive types). 
```

Comment:
Suggest: "Consideration **of** whether some..."

Reply:
Grammatical correctness of the bolding isn't desired. Re-reading it I want to shorten it further.